gocode.go

```diff
@@ -15,9 +15,9 @@ var (
 	g_addr                = flag.String("addr", "127.0.0.1:37373", "address for tcp socket")
 	g_debug               = flag.Bool("debug", false, "enable server-side debug mode")
 	g_source              = flag.Bool("source", false, "use source importer")
-	g_builtin             = flag.Bool("builtin", false, "propose builtin objects")
+	g_builtin             = flag.Bool("builtin", false, "propose completions for built-in functions and types")
+	g_unimported_packages = flag.Bool("stdlib", false, "propose completions for standard library packages not explicitly imported")
```

Comment:
```suggestion
	g_unimported_stdlib = flag.Bool("unimported-stdlib", false, "propose completions for standard library packages not explicitly imported")
```

If we rename the flag, we should also rename the corresponding variable. I would keep the "unimported" because I don't think "stdlib" is clear enough.